### DISCUSSION TITLE
Local plugin development

### DISCUSSION THREAD

**Discussion by openscript** [Js-Brecht/gatsby-plugin-ts-config#51] [Q&A] [answered]
Hello community

Is it possible that this issue is related to gatsby-plugin-ts-config?
https://github.com/gatsbyjs/gatsby/issues/33320

I'm a bit stuck here. Has someone an idea how to debug this further?

Thank you!

Robin

**Comment by Js-Brecht** [comment, marked as answer]
No, those modules are hit by Webpack during the build. `gatsby-plugin-ts-config` is entirely bootstrap (before the bootstrap even, if we’re being exact).

Also, this looks like an error generated by React itself, rather than a compiler generated error.

I would start by spinning it up in the debugger and breaking inside of `wrapPageElement` so you can see what `element` actually is (as well as the rest of the provided props in that first parameter).    That might give you a clue what is actually being provided to your function
